cmd/awscollector/Dockerfile

```diff
@@ -9,16 +9,13 @@ RUN apk --update add ca-certificates
 
 WORKDIR /workspace
 
-ARG TARGETPLATFORM
-RUN echo "Target platform is $TARGETPLATFORM"
```

Comment:
Think we should echo the target arch we are running for

Reply:
Wouldn't the TARGETARCH just show up in the COPY instruction when it tries to find the file?